src/features/header-flyout/HeaderFlyout.js

```diff
@@ -27,43 +29,44 @@ class HeaderFlyout extends React.Component<Props> {
 
     static defaultProps = {
         position: 'bottom-left',
+        isResponsive: false,
     };
 
     render() {
-        const { header, footer, flyoutButton, children, scrollRefFn, className, ...rest } = this.props;
+        const { header, footer, isResponsive, children, scrollRefFn, className, ...rest } = this.props;
 
         return (
-            <Flyout
+            <ResponsiveFlyout
                 closeOnClick={false}
                 offset={HeaderFlyout.panelOffset}
                 className={classNames('header-flyout', className)}
                 constrainToWindow
+                header={header}
+                overlayClassName="header-flyout-overlay"
+                enabled={isResponsive}
```

Comment:
If this is only used to set a class that depends on the FF, would we be able to remove this whole prop in favor using .is-responsive-web?